package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ton-community/contract-verifier-sdk",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
other than bumping the version, what was the change in this repo that caused index.min.js to change?